src/firefly/java/edu/caltech/ipac/firefly/server/query/ptf/PtfProcimsFileRetrieve.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+package edu.caltech.ipac.firefly.server.query.ptf;
+
+import edu.caltech.ipac.firefly.data.ServerRequest;
+import edu.caltech.ipac.firefly.server.query.SearchProcessorImpl;
+import edu.caltech.ipac.firefly.server.query.URLFileInfoProcessor;
+import edu.caltech.ipac.firefly.server.util.QueryUtil;
+import edu.caltech.ipac.util.StringUtils;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import static edu.caltech.ipac.firefly.server.query.ptf.PtfFileRetrieve.IBE_HOST;
+
+/**
+ * Created by IntelliJ IDEA. User: wmi Date: Feb 15, 2011 Time: 1:40:11 PM To change this template use File | Settings |
+ * File Templates.
+ */
+
+
+@SearchProcessorImpl(id = "PtfProcimsFileRetrieve")
```

Comment:
I've actually kept them because they are identical to the ife ones and i think they can be useful in the future. I just changed the `id` name to avoid confusion, but these classes are not for LC per se, they are for getting IBE PTF specifically.